auth/v1.js

```diff
@@ -816,6 +816,55 @@ api.declare({
   });
 });
 
+api.declare({
+  method:     'get',
+  route:      '/test-authenticate-get/:clientScope',
+  name:       'testAuthenticateGet',
+  output:     'test-authenticate-response.json#',
+  stability:  'experimental',
+  title:      "Test Authentication (GET)",
+  description: [
+    "Utility method similar to `testAuthenticate`, but with the GET method,",
+    "so it can be used with signed URLs (bewits).",
+    "",
+    "Rather than using real credentials, this endpoint accepts requests with",
+    "clientId `tester` and accessToken `no-secret`. That client's scopes are",
+    "`['test-authenticate-get', clientScope]`.  The call fails if the",
+    "`test-authenticate-get` scope is not avaialble (e.g,. via authenticateScopes)",
```

Comment:
If we really want to test the clients... we should make this use querystring for the two lists of scopes we had before...
Just say the querystring values have to JSON encoded lists of scopes. That should be easy to play with...

I suspect most of our client libraries either don't support querystring options yet, or don't have serious tests for them... And certainly not in combination with bewits...
We could say that if no query string is provided, then these hardcoded defaults are assumed.

That way the end-point can be used for testing both cases...


----
Okay, I guess in that way, we can just extend this end-point later... No need to do it now...

Note: query string parameters are always optional..